docker.go

```diff
@@ -284,14 +304,18 @@ func commandInfo() *exec.Cmd {
 }
 
 // helper function to create the docker build command.
-func commandBuild(build Build) *exec.Cmd {
+func commandBuild(build Build, builder Builder) *exec.Cmd {
```

Comment:
Do we need both Build and Buildx? In this repository, we should either replace Build -> Buildx or treat it as Buildx. Right?

Reply:
Build is for the build arguments and Builder is for the config for buildx builder.